pycqed/measurement/openql_experiments/generate_qi_cfg.py

```diff
@@ -54,13 +54,13 @@ def _rotation_instruction(phi, theta):
 
```

Comment:
Shouldn't generate config files be removed at all already now?

Reply:
No, we can remove this only once QINF is properly integrated.